angularjs-portal-home/src/main/webapp/my-app/layout/services.js

```diff
@@ -180,10 +181,22 @@ define(['angular', 'jquery'], function(angular, $) {
                         return data;
                     },
                     function(reason) {
-                        miscService.redirectUser(reason.status, 'exclusive markup for ' + portlet.fname + " failed.");
```

Comment:
We may still want to do the redirectUser function, that is in place in case the shib session is invalid so it redirects the user to the login page.